.circleci/config.yml

```diff
@@ -27,6 +27,14 @@ executors:
           - network.host: 127.0.0.1
           - http.port: 9200
           - discovery.type: single-node
+      - image: localstack/localstack:1.3.1
```

Comment:
think this will run this for all tests, how resource intensive is this?

Reply:
could try https://github.com/localstack/localstack-java-utils#usage for a class or two